### DISCUSSION TITLE
How to add coloured flavour text to items? + auto pick-up

### DISCUSSION THREAD

**Discussion by Danimal696** [flareteam/flare-game#850] [Q&A] [answered]
Can Flare add coloured flavour text to items?, started by WithinAmnesia in OGA:

https://opengameart.org/forumtopic/how-to-format-item-flavour-text-to-the-bottom#comment-

I will also add my own, can items besides money be auto-picked when near?

**Comment by dorkster** [comment, marked as answer]
Flavor text can be added to items with the `flavor` property. For example:

```
[item]
id=90
name=Krolan's Hammer
flavor=An essential tool for any blacksmith.
...
```

The color of the flavor text can be changed in `engine/font_colors.txt` by using the `item_flavor` property. The fantasycore mod uses this setting:

```
item_flavor=190,180,170
```

However, it is not possible to change the position of the flavor text. It is always displayed right after the item's name.

----

As for your other question, I think auto-pickup of other items would be a good addition. Mod authors should be able to define which items are eligible. Not sure if should be given separate options for currency vs other items. One general "Auto-Pickup" option might be enough.

**Reply by WithinAmesia** [reply]
Okay cool, thanks:-).
